app/components/Views/QRAccountDisplay/QRAccountDisplay.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import QRAccountDisplay from './index';
+import { fireEvent } from '@testing-library/react-native';
+import { renderScreen } from '../../../util/test/renderWithProvider';
+import initialBackgroundState from '../../../util/test/initial-background-state.json';
```

Comment:
```suggestion
import backgroundState from '../../../util/test/initial-root-state';
```

Not sure if the path is correct, but it would be awesome if we remove the `ts-expect-error` at this test file. And now for a better typing experience we are importing the backgroundState from the `initial-root-state`

Reply:
I agree it would be awesome, let's do it in a later interation

Reply:
I think using backgroundState from initial-root-state is in the scope as this is the new way to test with state.

Reply:
Just implemented it